docs/query-mode.md

```diff
@@ -0,0 +1,51 @@
+The Sarif.Multitool has a 'query' mode. Query mode takes a SarifLog and query expression and determines how many Results match the expression. It can write the count to the console, return the count as an exit code, signal if the count exceeds a threshold, or write an output log with the filtered results only.
```

Comment:
General comment: consider using backticks around code elements and commands, for example,

`Sarif.Multitool query In.sarif -e "RuleId = SM00251"`

`=`, `==`

Reply:
Good suggestion - fixed.